app/javascript/packs/otp-delivery-preference.js

```diff
@@ -24,12 +24,6 @@ const getOTPDeliveryMethods = () =>
 const isDeliveryOptionSupported = (delivery, selectedOption) =>
   selectedOption.getAttribute(`data-supports-${delivery}`) !== 'false';
 
-/**
```

Comment:
To limit scope of the pull request, I'm going to revert these changes, along with the conversion to `validated_form_for` for existing phone setup screens. We should revisit as part of the work in LG-5362.

Reply:
Reverted in d4ead5912. The changes there may prove useful for future reference.